src/pages/Loginpage/index.tsx

```diff
@@ -0,0 +1,85 @@
+import React  from 'react';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+import { Form } from 'react-bootstrap';
+import { Formik} from 'formik';
+import { initialValues } from 'pages/Loginpage/initialValues';
+import { validationSchema } from 'pages/Loginpage/validationSchema';
+import { Styled } from 'pages/Loginpage/Loginpage.styles';
+import CustomInputComponent from 'components/Input';
+import CustomButton from 'components/Button';
+import { login } from 'store/actions/auth';
+import { getAuthError, getIsAuthenticated, getAuthLoading } from 'store/selectors/auth';
+import { RootStore } from 'store';
+
+
+const Loginpage = ( props: any ) => {
+    const { loading, error, isAuthenticated } = props;
+
+    const submitHandler = (values: any) => {
+        props.loginHandler(values.username, values.password);
+    };
+
+    return(
+        <Styled.OuterContainer>
```

Comment:
Use OuterContainerWrapper or OuterWrapper here.

Word Wrapper at the end will automatically tells the reader that we have used a Styled component here.

Reply:
Used wrapper word for Styled components.